fetch/cosmwasm_contract/src/error.rs

```diff
@@ -0,0 +1,40 @@
+use cosmwasm_std::StdError;
+use thiserror::Error;
+
+/* *********************** *
+* Contract error messages *
+************************* *
+NB(LR) **don't** change, used as keywords by external code
+*/
+
+// limits error messages
+pub const ERR_SWAP_LIMITS_INCONSISTENT: &str =
+    "[FET_ERR_SWAP_LIMITS_INCONSISTENT] inconsistent swap fee and swap limits";
+pub const ERR_SWAP_LIMITS_VIOLATED: &str = "[FET_ERROR_SWAP_LIMITS] Swap limits violated";
+pub const ERR_SUPPLY_EXCEEDED: &str = "[FET_ERR_SUPPLY_EXCEEDED] Amount would exceed supply";
+pub const ERR_RA_ALLOWANCE_EXCEEDED: &str =
+    "[FET_ERR_RA_ALLOWANCE_EXCEEDED] Amount would exceed reverse aggregated allowance";
+pub const ERR_CAP_EXCEEDED: &str = "[FET_ERR_CAP_EXCEEDED] Amount would exceed cap";
+// access control error messages
+pub const ERR_ACCESS_CONTROL: &str = "[FET_ERR_ACCESS_CONTROL] ";
+pub const ERR_ACCESS_CONTROL_ONLY_ADMIN: &str = "[FET_ERR_ACCESS_CONTROL] Only Admin";
+pub const ERR_ACCESS_CONTROL_ONLY_RELAYER: &str = "[FET_ERR_ACCESS_CONTROL] Only Relayer";
+pub const ERR_ACCESS_CONTROL_ALREADY_HAVE_ROLE: &str = "[FET_ERR_ACCESS_CONTROL] Already have role";
+pub const ERR_ACCESS_CONTROL_DONT_HAVE_ROLE: &str = "[FET_ERR_ACCESS_CONTROL] Don't have role";
```

Comment:
Just nit picking (not neccessary to change): DONT -> DIESN't ansd don't - > doesn't